src/control.py

```diff
@@ -45,61 +106,38 @@ def init_i2c(self):
 
     def init_leak(self):
         self.leak_sensor= AnalogIn(self.ads,ADS.P0)
-        
-    def water_level_init(self):
-        self.level = 0
-        self.setup()
-        self.read() # update level 
-         
          
     def init_ph(self):
         self.pH_sensor= AnalogIn(self.ads,ADS.P1)
+
+    def init_pH(self):
+        self.pH_sensor= AnalogIn(self.ads,ADS.P1)
        
     def read_leak(self):
         return self.leak_sensor.voltage
 
     def read_pH(self):
         pH_voltage = self.pH_sensor.voltage
-        pH = 7.7 +(pH_voltage-14.7/10)*(-3.3) #formula adjusted for use with a voltage divider to map the 5 V output to 3.3V for use with a 3.3V ADC
+        pH = 7.7 +(pH_voltage-1.65)*(-3.3)
         return pH
-        
-    def read_waterlevel(self):
-        try:
-            #self.level = GPIO.input(pins.WATER_LEVEL)
-            self.level = GPIO.input(11)
-            return self.level
-        except:
-            GPIO.cleanup()        
-            return -1
-        
-    def pump_open(self):
-        print ('pump opened')
-        GPIO.output(9,GPIO.HIGH) #9 for the relay pin on pH pump
-        time.sleep(1)
-        GPIO.output(9,GPIO.LOW)
-    #Here I add the function of the timer
-    #This code here correspond update the data at default time
-    #Using threading method
-    def test_ph(self):
+
+    def read_ph(self):
+        pH_voltage = self.pH_sensor.voltage
+        pH = 7.7 +(pH_voltage-1.65)*(-3.3)
+        return pH
+		
+    def test_wl(self):
```

Comment:
So test_wl(self) is the control loop for the water level?

Reply:
Yes exactly!

Reply:
Great! If you can just add a brief description, so that future users know what this module's function is.